dotCMS/src/main/java/com/dotcms/security/apps/AppSecretSavedEvent.java

```diff
@@ -0,0 +1,41 @@
+package com.dotcms.security.apps;
+
+import com.dotmarketing.beans.Host;
+import java.io.Serializable;
+
+/**
+ * AppSecretSavedEvent
+ * Broadcast when a secret is saved.
+ */
+public class AppSecretSavedEvent implements Serializable {
```

Comment:
Codacy found an issue: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5856383)